packages/shared/src/graphql/sources.ts

```diff
@@ -52,6 +56,15 @@ export interface Squad extends Source {
   referralUrl?: string;
 }
 
+export interface BasicSquadWithCurrentMember extends Partial<Squad> {
+  membersCount: number;
+  permalink: string;
+  id: string;
+  name: string;
+  currentMember?: SourceMember &
+    Required<Pick<SourceMember, 'referralToken' | 'permissions'>>;
```

Comment:
Why did we end up doing a Partial of the Squad. All of the properties seem to be existing already from the `Squad` type.